stories/checkbox.js

```diff
@@ -4,12 +4,17 @@ import { storiesOf } from '@storybook/react';
 import { checkA11y } from 'storybook-addon-a11y';
 import { withInfo } from '@storybook/addon-info';
 import { withKnobs, boolean, select, text } from '@storybook/addon-knobs/react';
-import { Checkbox, Link, TextBody } from '../components';
+import { Checkbox, Link, TextBody, TextDisplay, TextSmall } from '../components';
+import { Text } from 'glamorous';
+import { textFactory } from '../components/typography/Text';
```

Comment:
Q: Why exactly are you importing these last 2?

Reply:
I think this must have been added automatically by a plugin I used in my editor. Do I just remove this in a new commit?

Reply:
@InstaK I removed the unnecessary imports from the checkbox story